cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1086,6 +1117,24 @@ public Secret generateBrokersSecret() {
         return createSecret(KafkaCluster.brokersSecretName(cluster), data);
     }
 
+    /**
+     * Generate the Secret containing the username and password to secure the jmx port on the kafka brokers
+     *
+     * @return The generated Secret
+     */
+    public Secret generateJmxSecret() {
+
+        Map<String, String> data = new HashMap<>();
+        String[] keys = {SECRET_JMX_USERNAME_KEY, SECRET_JMX_PASSWORD_KEY};
+        for (String key : keys) {
+            String value = UUID.randomUUID().toString();
```

Comment:
we have a `PasswordGenerator` class for this kind of things, maybe we can use that in this case as well?

Reply:
I didn't know that you had one, I will use that instead!